src/extremeweatherbench/defaults.py

```diff
@@ -0,0 +1,236 @@
+from extremeweatherbench import derived, inputs, metrics
+
+# ERA5 targets
+era5_heatwave_target = inputs.ERA5(
+    source=inputs.ARCO_ERA5_FULL_URI,
+    variables=["surface_air_temperature"],
+    variable_mapping={
+        "2m_temperature": "surface_air_temperature",
+        "time": "valid_time",
+    },
+    storage_options={"remote_options": {"anon": True}},
+)
+
+era5_freeze_target = inputs.ERA5(
+    source=inputs.ARCO_ERA5_FULL_URI,
+    variables=[
+        "surface_air_temperature",
+        "surface_eastward_wind",
+        "surface_northward_wind",
+    ],
+    variable_mapping={
+        "2m_temperature": "surface_air_temperature",
+        "10m_u_component_of_wind": "surface_eastward_wind",
+        "10m_v_component_of_wind": "surface_northward_wind",
+        "time": "valid_time",
+    },
+    storage_options={"remote_options": {"anon": True}},
+)
+
+era5_atmospheric_river_target = inputs.ERA5(
+    source=inputs.ARCO_ERA5_FULL_URI,
+    variables=[
+        derived.IntegratedVaporTransport,
+        derived.AtmosphericRiverMask,
+    ],
+    variable_mapping={
+        "u_component_of_wind": "eastward_wind",
+        "v_component_of_wind": "northward_wind",
+        "specific_humidity": "specific_humidity",
+        "temperature": "air_temperature",
+        "vertical_integral_of_northward_water_vapour_flux": "northward_water_vapour_flux",
+        "vertical_integral_of_eastward_water_vapour_flux": "eastward_water_vapour_flux",
+    },
+    storage_options={"remote_options": {"anon": True}},
+)
+
+# GHCN targets
+ghcn_heatwave_target = inputs.GHCN(
+    source=inputs.DEFAULT_GHCN_URI,
+    variables=["surface_air_temperature"],
+    variable_mapping={"t2": "surface_air_temperature"},
+    storage_options={"remote_protocol": "s3", "remote_options": {"anon": True}},
+)
+
+ghcn_freeze_target = inputs.GHCN(
+    source=inputs.DEFAULT_GHCN_URI,
+    variables=[
+        "surface_air_temperature",
+        "surface_eastward_wind",
+        "surface_northward_wind",
+    ],
+    variable_mapping={
+        "surface_temperature": "surface_air_temperature",
+        "surface_eastward_wind": "surface_eastward_wind",
+        "surface_northward_wind": "surface_northward_wind",
+    },
+    storage_options={"remote_options": {"anon": True}},
+)
+
+# LSR/PPH target
+
+# TODO: add PPH target
+lsr_target = inputs.LSR(
+    source=inputs.LSR_URI,
+    variables=["local_storm_reports"],
+    variable_mapping={},
+    storage_options={"remote_options": {"anon": True}},
+)
+
+# IBTrACS target
```

Comment:
I'm an outsider. I have no idea what this means, but I assume this has to do with TC tracks. If it were possible to make this a general descriptive name, I would be all for it.